src/adapters/destination-adapters/ChronicleMedianDestinationAdapter.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
do we want mixing licenses? this repo currently has BUSL-1.1

Reply:
@Reinis-FRP you're right. it should be the same license that this repo currently has.